lib/intl.mjs

```diff
@@ -107,8 +107,8 @@ export function DateTimeFormat(locale = undefined, options = undefined) {
   this[INST] = instantAmend;
 }
 
-DateTimeFormat.supportedLocalesOf = function (...args) {
-  return IntlDateTimeFormat.supportedLocalesOf(...args);
+DateTimeFormat.supportedLocalesOf = function (locales, options) {
```

Comment:
This change seems unrelated, but makes sense to me, because it makes `Intl.DateTimeFormat.supportedLocalesOf.length` conformant.

Reply:
I changed this because tsc doesn't like using the argument spread to invoke supportedLocalesOf, but making it length conformant is handy :)

Reply:
@ptomato - does that mean that the proposal-temporal polyfill is *not* conformant here, and if so should this change be ported over there?

Reply:
Yes, but:
1. Intl.DateTimeFormat.supportedLocalesOf is not part of the Temporal proposal, so it won't be tested and probably doesn't matter. For this polyfill, it does matter, because people might replace their Intl object in production code.
2. I also realized it is still not conformant. This makes it have length 2 rather than 0, but it is actually supposed to have length 1. The signature is Intl.DateTimeFormat.supportedLocalesOf ( locales [ , options ] )

Reply:
@ptomato - so the right signature would be something like `function (locales, options = {})` ?

@12wrigja - I'd suggest splitting this change out into a new PR, and getting the conformance correct.

Reply:
Oops, just noticed this is already merged! Now we definitely need a separate PR to fix the conformance. ;-)